sdks/python/apache_beam/utils/urns.py

```diff
@@ -68,14 +69,14 @@ class RunnerApiFn(object):
 
   _known_urns = {}  # type: Dict[str, Tuple[Optional[type], ConstructorFn]]
 
-  @abc.abstractmethod
+  # @abc.abstractmethod
```

Comment:
Please add a comment here as to why (mypy vs. dynamic patching).

Reply:
done.